tests/functional/disaster-recovery/regional-dr/conftest.py

```
@@ -449,6 +450,70 @@ def teardown():
+
+    The factory creates the storage class and pool on each cluster if they do
+    not already exist, then resets the cluster context.
+
+    Raises:
+        Exception: If storage class creation fails on any of the managed clusters
+
+    Returns:
+        callable: factory(replica, compression) that creates the custom SC
+    """
+
+    def factory(replica, compression, erasure_coded=False):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `factory`

Add return type annotation: `None`

(ANN202)